pkg/manilaapi/statefulset.go

```diff
@@ -70,6 +72,46 @@ func StatefulSet(
 			Port: intstr.IntOrString{Type: intstr.Int, IntVal: int32(manila.ManilaPublicPort)},
 		}
 		readinessProbe.HTTPGet = livenessProbe.HTTPGet
+
+		if instance.Spec.TLS.API.Enabled(service.EndpointPublic) {
+			livenessProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+			readinessProbe.HTTPGet.Scheme = corev1.URISchemeHTTPS
+		}
+	}
+
+	// create Volume and VolumeMounts
+	volumes := GetVolumes(manila.GetOwningManilaName(instance), instance.Name, instance.Spec.ExtraMounts)
```

Comment:
this could be just 
```
	volumes := append(GetVolumes(manila.GetOwningManilaName(instance), instance.Name, instance.Spec.ExtraMounts),  GetLogVolume()...)
```

then logVolume won't be required?